charts/argoconfig/values.yaml

```diff
@@ -28,38 +40,34 @@ appProject:
   sourceRepos:
     - '*'
 
-application:
-  # application.enabled -- Enable creating an Application resource
-  enabled: true
-  # application.namespace -- Namespace for Application resource
-  namespace: argocd
-  # application.projectOverride -- Name of AppProject to install into
-  # @default -- Fullname
-  projectOverride: ~
-  source:
-    # application.source.repoURL -- URL of sourece repo
-    repoURL: ""
-    # application.source.chart -- Name of chart in source repo
-    chart: ""
-    # application.source.targetRevision -- Revision of chart in source repo to install
-    targetRevision: '*'
-  destination:
-    # application.destination.server -- Target cluster in Argo CD
-    server: https://kubernetes.default.svc
-    # application.destination.namespace -- Target namespace of application in Argo CD
-    namespace: default
-  # application.syncPolicy -- Application [Sync Policy](https://argoproj.github.io/argo-cd/user-guide/auto_sync/)
-  # @default -- deactivated
-  syncPolicy: {}
-
-cluster:
-  # cluster.enabled -- Create a cluster config secret
-  enabled: false
-  # cluster.clusterNameOverride -- Override name of the cluster
-  clusterNameOverride: ~
-  # cluster.namespace -- Namespace to put the cluster config secret into
-  namespace: argocd
-  # cluster.server -- Cluster URL used by Argo CD to connect to the cluster
-  server: https://kubernetes.default.svc
-  # cluster.config -- Cluster config (ie. contents of a KUBECONFIG)
-  config: {}
+apps:
+  example:
+    # apps.example.enabled -- Enable creating an Application resource
+    enabled: false
+    # apps.example.name -- Name of app, used as suffix to app specific
+    # resources
+    name: "false"
```

Comment:
can't we define the name of the example app "example"?

Reply:
is `$app.name` even used somewhere?

Reply:
I got rid of that when I refactored apps to a map instead of an array and should get rid of it here as well.